odh/overlays/moc/zero/jupyterhub/obcs/user-yaar-obc.enc.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
+metadata:
+    annotations:
+        username: ENC[AES256_GCM,data:LB/dc6oVydV45vnhTqrH3uA=,iv:WLH9r4KASFgADmHpuBe667e+K3mU3n3ATi/WaE8vBPw=,tag:tI+eez2+P+eqSxi2CUkGhw==,type:str]
+    name: ENC[AES256_GCM,data:YV8tIcph1dEsuHoTid+YtfnrgcY=,iv:j1dtM46voGRr5DTpqbI0ihQLXXLS2GESKHcw399CrRA=,tag:vSimDpHzBpFBo2Hb5hFGKQ==,type:str]
+spec:
+    generateBucketName: ENC[AES256_GCM,data:679f2woRKGoOnXqADlzKr77AvYHp,iv:vsMyUU+SC2s+WSwCsmOL7SGHgxTFX6NJZgewwvj2p20=,tag:5g5oSfM5dhAldl9mu1DWkw==,type:str]
```

Comment:
@HumairAK I think we can just use `bucketName` since this is based on a username so we don't have to worry about a duplicate bucket

Reply:
are you proposing in this case we should just have the bucket-name be that of the requester?

Reply:
Yes